corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserMain.java

```diff
@@ -56,6 +59,8 @@
         + "--numItems=<numItems> Total Number of items for loadTable\n"
         + "--batchSize=<batchSize> Number of records per transaction for loadTable\n"
         + "--itemSize=<itemSize> Size of each item's payload for loadTable\n"
+        + "--keyToEdit=<keyToEdit> Key of the record to edit\n"
+        + "--newRecord=<newRecord> New Editted record to insert\n"
```

Comment:
Edit key and add key almost share the same logic, why not support both ?

Reply:
valid point

Reply:
I created a new issue https://github.com/CorfuDB/CorfuDB/issues/2992 for it and plan to address it there because addRecord also needs metadata as an input.